### ISSUE TITLE
imagemin memory usage too high

### ISSUE THREAD

**Issue by nusability** [gruntjs/grunt-contrib-imagemin#200]
A build task can fail on a reasonably modest machine that has some services like mysql running.

Example: I am developing on a virtual machine that has 2gb ram and hosts a couple of services like mysql and email servers. A 'grunt build' task even for a small web app can fail at the imagemin-task without a proper error message, presumably because the process runs out of memory to allocate. 'grunt build --verbose' doesn't display an error either but shows that the task fails at imagemin.

Stopping services like mysql, php-fpm and such solved my immediate problem but is an unsatisfying solution.

Hope this helps!

**Comment by ArtemFitiskin** [comment]
Today there was a similar case on my production server. It so happened that I had to re-install npm deps for my application. Specifically, grunt-contrib-imagemin download process directly at the end (during the pre-build test) stops responding. No any error in the console. Dozens of PID eaten all free RAM (~6-8 gigabytes). On the server running many services using crontab and mailings. When I did this the last time these services were stop. Now I do not have this capability. On the local machine with a similar configuration such never happened.

grunt-contrib-imagemin v0.9.2
npm v1.3.6
Node v0.10.33
CentOS 6.5
